features/step_definitions/oauth.rb

```diff
@@ -17,3 +17,11 @@
     sleep 60
   }
 end
+
+# extract the secret given a htpasswd spec name
+Given /^the secret for #{QUOTED} htpasswd is stored in the#{OPT_SYM} clipboard$/ do |name, cb_name|
+  cb_name ||= :htpasswd_secret
+  generated_htpasswd_name = o_auth('cluster').generated_htpasswd_name(name: name)
+  step %Q/I use the "openshift-config" project/
```

Comment:
Do we need to change current project? Can we avoid this?

Reply:
looks like we can, I'll take that line out